tests/storage-volumes-vm

```diff
@@ -99,6 +99,11 @@ do
 	lxc storage volume attach "${poolName}" vol2 v1
 	lxc storage volume attach "${poolName}" vol3 v1
 
+	echo "==> Share a custom block volume"
+	! lxc storage volume attach "${poolName}" vol2 v2 || false
+	lxc storage volume set default vol2 security.shared true
+	lxc storage volume attach "${poolName}" vol2 v2
```

Comment:
You might also want to detach `vol2` from `v2` (and reset `security.shared`?) as part of the cleanup down below.

It doesn't really matter as both the volume and instance get removed afterwards but if you add a test case in the future it's then easier to understand what is the current state of the entities so you don't accidentally make tests with an instance or volume in a state that you don't expect it to be in.

Reply:
should I use the oportunity to detach the other volumes from `v2`?

Reply:
You mean removing `vol5` and `vol6` which are getting attached below?
As `v2` gets removed anyway after those tests I guess its fine like this.

But attaching the same vol twice feels niche so just unsetting `security.shared` and detaching it again from v2 would feel right in my opinion. You could do this right at the end of the test you have added.
With this you make sure the test doesn't leave any traces behind.